src/operations/map/MapOperations.ts

```diff
@@ -9,14 +10,40 @@ import { GlobalInsight } from '@/domain/enums/GlobalInsight.ts';
 import { MapProps } from '@/domain/props/MapProps';
 import { getColors } from '@/styles/MapColors.ts';
 
+import { IPCMapOperations } from '../IPC/IpcMapOperations';
+
 export class MapOperations {
   static createMapboxMap(
     isDark: boolean,
-    { countries }: MapProps,
+    { countries, ipcData, selectedMapType }: MapProps,
     mapContainer: RefObject<HTMLDivElement>
   ): mapboxgl.Map {
     const mapColors: MapColorsType = getColors(isDark);
 
+    const getCountryFillPaint = (): DataDrivenPropertyValueSpecification<string> => {
```

Comment:
maybe `getCountryFillColor`?

Reply:
also categorize it as `getCountryFillColorForIPC`

Reply:
yeah this is definitely more descriptive